test/REQUIRE

```diff
@@ -0,0 +1 @@
+Compat
```

Comment:
doing this is a test dependency will mask the fact that codeunits and uninitialized are used in src but not present in 0.6

Reply:
Doy, of course.  Thanks.